2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Current SDK version is 0.15.1
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Configure stats pid to 3484
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Anton\.config\wandb\settings
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Loading settings from f:\Glaucoma\wandb\settings
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-03 16:49:39,639 INFO    MainThread:3484 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:_log_setup():507] Logging user logs to f:\Glaucoma\wandb\run-20230503_164939-0x001\logs\debug.log
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:_log_setup():508] Logging internal logs to f:\Glaucoma\wandb\run-20230503_164939-0x001\logs\debug-internal.log
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002BCA4E61C70>
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:init():547] calling init triggers
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'Dataset': 'Glaucoma', 'Models': 'Xception', 'learning_rate': 0.001, 'beta1': 0.9, 'metrics': ['accuracy']}
2023-05-03 16:49:39,640 INFO    MainThread:3484 [wandb_init.py:init():596] starting backend
2023-05-03 16:49:39,641 INFO    MainThread:3484 [wandb_init.py:init():600] setting up manager
2023-05-03 16:49:39,644 INFO    MainThread:3484 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-05-03 16:49:39,651 INFO    MainThread:3484 [wandb_init.py:init():606] backend started and connected
2023-05-03 16:49:39,667 INFO    MainThread:3484 [wandb_run.py:_label_probe_notebook():1238] probe notebook
2023-05-03 16:49:39,669 INFO    MainThread:3484 [wandb_run.py:_label_probe_notebook():1248] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-05-03 16:49:39,669 INFO    MainThread:3484 [wandb_init.py:init():700] updated telemetry
2023-05-03 16:49:39,688 INFO    MainThread:3484 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-03 16:49:40,900 INFO    MainThread:3484 [wandb_init.py:init():780] run resumed
2023-05-03 16:49:40,908 INFO    MainThread:3484 [wandb_run.py:_on_init():2177] communicating current version
2023-05-03 16:49:41,241 INFO    MainThread:3484 [wandb_run.py:_on_init():2186] got version response 
2023-05-03 16:49:41,241 INFO    MainThread:3484 [wandb_init.py:init():787] starting run threads in backend
2023-05-03 16:49:41,346 INFO    MainThread:3484 [wandb_run.py:_console_start():2158] atexit reg
2023-05-03 16:49:41,347 INFO    MainThread:3484 [wandb_run.py:_redirect():2013] redirect: SettingsConsole.WRAP_RAW
2023-05-03 16:49:41,347 INFO    MainThread:3484 [wandb_run.py:_redirect():2078] Wrapping output streams.
2023-05-03 16:49:41,347 INFO    MainThread:3484 [wandb_run.py:_redirect():2103] Redirects installed.
2023-05-03 16:49:41,347 INFO    MainThread:3484 [wandb_init.py:init():829] run started, returning control to user process
2023-05-03 16:49:41,351 INFO    MainThread:3484 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-03 16:49:41,352 INFO    MainThread:3484 [wandb_init.py:_pause_backend():418] pausing backend
2023-05-03 16:49:41,453 INFO    MainThread:3484 [wandb_init.py:_resume_backend():423] resuming backend
2023-05-03 16:49:41,457 INFO    MainThread:3484 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-03 16:49:41,457 INFO    MainThread:3484 [wandb_init.py:_pause_backend():418] pausing backend
2023-05-03 16:49:41,529 INFO    MainThread:3484 [wandb_init.py:_resume_backend():423] resuming backend
2023-05-03 16:49:41,532 INFO    MainThread:3484 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-03 16:49:41,532 INFO    MainThread:3484 [wandb_init.py:_pause_backend():418] pausing backend
2023-05-03 16:49:41,605 INFO    MainThread:3484 [wandb_init.py:_resume_backend():423] resuming backend
2023-05-03 16:49:44,417 INFO    MainThread:3484 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-03 16:49:44,417 INFO    MainThread:3484 [wandb_init.py:_pause_backend():418] pausing backend
2023-05-03 16:49:44,471 INFO    MainThread:3484 [wandb_init.py:_resume_backend():423] resuming backend
2023-05-03 16:50:09,182 INFO    MainThread:3484 [wandb_run.py:_finish():1893] finishing run webcoders/Glaucoma Detection/0x001
2023-05-03 16:50:09,331 INFO    MainThread:3484 [jupyter.py:save_history():445] not saving jupyter history
2023-05-03 16:50:09,331 INFO    MainThread:3484 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-03 16:50:09,331 INFO    MainThread:3484 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-05-03 16:50:09,331 INFO    MainThread:3484 [wandb_run.py:_atexit_cleanup():2127] got exitcode: 0
2023-05-03 16:50:09,332 INFO    MainThread:3484 [wandb_run.py:_restore():2110] restore
2023-05-03 16:50:09,332 INFO    MainThread:3484 [wandb_run.py:_restore():2116] restore done
2023-05-03 16:50:16,140 INFO    MainThread:3484 [wandb_run.py:_footer_history_summary_info():3469] rendering history
2023-05-03 16:50:16,140 INFO    MainThread:3484 [wandb_run.py:_footer_history_summary_info():3501] rendering summary
2023-05-03 16:50:16,148 INFO    MainThread:3484 [wandb_run.py:_footer_sync_info():3428] logging synced files
